packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json

```diff
@@ -2,8 +2,9 @@
   "sms_section_back_link": "Revenir aux options",
   "sms_options_recredit_title": "Recharge automatique",
   "sms_options_recredit_information_title": "Informations",
-  "sms_options_recredit_information_text": "Vous avez la possibilité de recharger automatiquement votre compte SMS lorsque votre solde franchit un certain seuil.<br/>Cette recharge, qui s'effectue automatiquement, sera prélevée sur votre compte client OVH.",
-  "sms_options_recredit_information_note": "Veuillez noter que la recharge ne sera possible que si la balance de votre compte client OVH est positive ou nulle. Il vous est possible de créditer votre compte client dans votre Espace Client Télécom section Facturation.",
+  "sms_options_recredit_information_text": "Vous avez la possibilité de recharger automatiquement votre compte SMS lorsque votre solde franchit un certain seuil.",
+  "sms_options_recredit_information_note": "L’option de recharge automatique ne peut être activée qu'à partir de 2 mois d'ancienneté de votre service. Un moyen de paiement de type Prélèvement SEPA doit également être présent et valide sur votre compte OVHcloud.",
```

Comment:
```suggestion
  "sms_options_recredit_information_note": "L’option de recharge automatique ne peut être activée qu'à partir de 2 mois d'ancienneté de votre service. Un moyen de paiement de type prélèvement SEPA doit également être présent et valide sur votre compte OVHcloud.",
```

Reply:
@mohammed-zahaf, `Prélèvement SEPA` should be kept as a title form (capitalized). See for example: https://github.com/ovh/manager/commit/489b27bfd714edd65595b833780da751b9169899#diff-a52bb7ff0674c759d0861b68df01d40388d1938ad676fd81454b6bc3020dca67R2